### ISSUE TITLE
Question: Why is the hybrid simulator suggested to use max_order=3

### ISSUE THREAD

**Issue by Jconn** [LCAV/pyroomacoustics#256]
Hello,

I've been using the the hybrid simulator for generating my RIR and have noticed different results if I vary my max_order from small to large values. I see in the documentation the following recommendation `We suggest to use max_order=3 with the hybrid simulator.`

But within that same documentation I see instances of `max_order=17` being used with ray_tracing enabled. I was looking for an explanation of why `max_order=3` is recommended but I couldn't find it in the documentation. 

Why is max_order=3 recommended for the hybrid simulator?

**Comment by fakufaku** [comment]
Hi, thanks for the question! The motivation for this statement is that the image source model (ISM) models well specular reflections while ray tracing handles the diffuse part of the RIR.
It happens that the early echoes are responsible for most of the specular reflections so that limiting the ISM to a low order is reasonable (when used together with ray tracing).
The following figure illustrates the concept.
<img width="1069" alt="image" src="https://user-images.githubusercontent.com/1118133/160290752-f9ed7bba-1bad-4ccb-a245-6960c7d2f6a0.png">
Source: Dirk Schröder, Physically Based Real-Time Auralization of Interactive Virtual Environments, PhD Thesis, 2011.

**Comment by Jconn** [comment]
Thanks! That's very helpful. I have been running into an issue where when I enable ray tracing and set my ISM order to something large(>50), the reverberation in my signal doesn't appear diffuse. Does setting a high `max_order` lock out the ray tracing for those reflections?

**Comment by fakufaku** [comment]
Yes, it does! If you don't use scattering materials, the ray tracing part will be ignored up to max_order of the rays.
If you do use scattering, the scattered rays from ray tracing will be accounted for, since ISM does not model that.
